backend/src/model/entity.rs

```diff
@@ -11,70 +11,230 @@ use diesel::{Identifiable, Insertable, Queryable};
-    soil_water_retention::SoilWaterRetention, sun::Sun, water::Water,
+    deciduous_or_evergreen::DeciduousOrEvergreen, external_source::ExternalSource,
+    fertility::Fertility, flower_type::FlowerType, growth_rate::GrowthRate,
+    herbaceous_or_woody::HerbaceousOrWoody, life_cycle::LifeCycle,
+    light_requirement::LightRequirement, nutrition_demand::NutritionDemand,
+    propagation_method::PropagationMethod, quality::Quality, quantity::Quantity, soil_ph::SoilPh,
+    soil_texture::SoilTexture, soil_water_retention::SoilWaterRetention,
+    water_requirement::WaterRequirement,
 };
 
 /// The `Plants` entity.
 #[allow(clippy::missing_docs_in_private_items)] // TODO: See #97.
 #[derive(Identifiable, Queryable)]
 #[diesel(table_name = plants)]
 pub struct Plants {
+    /// The internal id of the plant.
     pub id: i32,
-    pub binomial_name: String,
-    pub common_name: Option<Vec<Option<String>>>,
+    /// The unique name of the plant. The structure is described in the /doc/database/hierarchy.md document in the repository.
+    pub unique_name: String,
+    /// The list of the common names of the plant.
+    pub common_name_en: Option<Vec<Option<String>>>,
+    /// The list of the common names of the plant in german. Fetched from Wikidata API if not present in the source datasets.
     pub common_name_de: Option<Vec<Option<String>>>,
+    /// Family of the plant.
     pub family: Option<String>,
+    /// Subfamily of the plant.
     pub subfamily: Option<String>,
+    /// Genus of the plant.
     pub genus: Option<String>,
-    pub edible_uses: Option<String>,
+    ///
+    pub edible_uses_en: Option<String>,
+    ///
     pub medicinal_uses: Option<String>,
+    ///
     pub material_uses_and_functions: Option<String>,
+    ///
     pub botanic: Option<String>,
-    pub propagation: Option<String>,
-    pub cultivation: Option<String>,
-    pub environment: Option<String>,
+    ///
     pub material_uses: Option<String>,
+    ///
     pub functions: Option<String>,
-    pub provides_forage_for: Option<String>,
-    pub provides_shelter_for: Option<String>,
-    pub hardiness_zone: Option<i16>,
+    ///
     pub heat_zone: Option<i16>,
-    pub water: Option<Water>,
-    pub sun: Option<Sun>,
+    ///
     pub shade: Option<String>,
+    /// The list of enum values.
     pub soil_ph: Option<Vec<Option<SoilPh>>>,
+    /// The list of enum values.
     pub soil_texture: Option<Vec<Option<SoilTexture>>>,
+    /// The list of enum values.
     pub soil_water_retention: Option<Vec<Option<SoilWaterRetention>>>,
+    ///
     pub environmental_tolerances: Option<Vec<Option<String>>>,
-    pub native_climate_zones: Option<String>,
-    pub adapted_climate_zones: Option<String>,
+    ///
     pub native_geographical_range: Option<String>,
+    ///
     pub native_environment: Option<String>,
+    ///
     pub ecosystem_niche: Option<String>,
-    pub root_zone_tendancy: Option<RootZoneTendancy>,
+    /// Enum value.
```

Comment:
Please remove such comments (it can be seen from the type anyway). Important is which columns you fetched from where, some non-obvious mapping you might have done etc.